PurchasesTests/Mocks/MockStoreKit2TransactionListener.swift

```diff
@@ -60,7 +60,7 @@ class MockStoreKit2TransactionListener: StoreKit2TransactionListener {
     var invokedHandleParameters: (purchaseResult: Any, Void)?
     var invokedHandleParametersList = [(purchaseResult: Any, Void)]()
 
-    override func handle(purchaseResult: StoreKit.Product.PurchaseResult) async -> Bool {
+    override func handle(purchaseResult: StoreKit.Product.PurchaseResult) async throws -> Bool {
```

Comment:
fun times

Reply:
Do we want to add a comment on this mentioning that bug and that `throws` is required until Swift `5.6` is our minimum?

I _could_ see a situation where this gets removed on accident on the future because it doesn't look like it needs to be there 🤷‍♂️

Reply:
It doesn't hurt to have it since it's inheriting a method with `throws`. The tests crash without it so it's not like we wouldn't notice.
But I can leave a comment to remove it on Swift 5.6.